include/pushmi/flow_single.h

```diff
@@ -32,8 +32,9 @@ class flow_single<V, PE, E> {
     void (*value_)(data&, V) = s_value;
     void (*stopping_)(data&) noexcept = s_stopping;
     void (*starting_)(data&, any_none<PE>&) = s_starting;
-    PUSHMI_DECLARE_CONSTEXPR_IN_CLASS_INIT(static vtable const noop_);
-  } const* vptr_ = &vtable::noop_;
+  };
+  PUSHMI_DECLARE_CONSTEXPR_IN_CLASS_INIT(static vtable const noop_);
+  vtable const* vptr_ = &noop_;
```

Comment:
@kirkshoop I wonder if `PUSHMI_DECLARE_CONSTEXPR_IN_CLASS_INIT` is still necessary for nvcc now that we are no longer trying to initialize an object of incomplete type.

Reply:
Yes, I expect that moving it outside the vtable struct will allow those macros to be removed